test/e2e/play_kube_test.go

```diff
@@ -293,13 +296,14 @@ type HostAlias struct {
 // if no containers are added, it will add the default container
 func getPod(options ...podOption) *Pod {
 	p := Pod{
-		Name:        defaultPodName,
-		Hostname:    "",
-		HostAliases: nil,
-		Ctrs:        make([]*Ctr, 0),
-		Volumes:     make([]*Volume, 0),
-		Labels:      make(map[string]string),
-		Annotations: make(map[string]string),
+		Name:          defaultPodName,
+		RestartPolicy: "",
```

Comment:
we may want the default here to be `never` as that matches the behavior the tests were written to expect

Reply:
got it